webeoc/src/staging-complaints-api-service/staging-complaints-api-service.service.ts

```diff
@@ -19,39 +18,35 @@ export class StagingComplaintsApiService {
       // Include the API key in the request headers
       const config = {
         headers: {
-          'x-api-key': process.env.COMPLAINTS_API_KEY, // Ensure this environment variable is set
+          "x-api-key": process.env.COMPLAINTS_API_KEY, // Ensure this environment variable is set
         },
       };
 
       const response = await axios.post(apiUrl, complaintData, config);
       this.logger.debug(`Staging Complaint API Response: ${response.data}`);
-      this.complaintsPublisherService.publishStagingComplaintInserted(
-        complaintData.incident_number,
-      );
+      this.complaintsPublisherService.publishStagingComplaintInserted(complaintData.incident_number);
     } catch (error) {
-      this.logger.error('Error calling Staging Complaint API:', error);
+      this.logger.error("Error calling Staging Complaint API:", error);
     }
   }
 
+  // create complaint based on complaint data in the staging table
   async postComplaint(complaint_identifier: string): Promise<void> {
     try {
-      this.logger.debug(
-        'Creating new complaint based on new complaint from webeoc.',
-      );
+      this.logger.debug("Creating new complaint based on new complaint from webeoc.");
       const apiUrl = `${process.env.COMPLAINTS_MANAGEMENT_API_URL}/staging-complaint/process/${complaint_identifier}`;
       this.logger.debug(`Posting new complaint. API URL: ${apiUrl}`);
 
       // Include the API key in the request headers
       const config = {
```

Comment:
I'd recommend turning this: 
```
      const config = {
        headers: {
          'x-api-key': process.env.COMPLAINTS_API_KEY, // Ensure this environment variable is set
        },
      };
```
into this:
```
  private readonly _apiConfig = { 
    headers: { 
      "x-api-key": process.env.COMPLAINTS_API_KEY
    }
  }
``` 
into a reusable private object as you're using this object in a few other locations. in the event that you need to add a header, or additional config value you can spread your source config object into a new local config.

Reply:
Good find, I'll make that change.

Reply:
Fixed